### PR TITLE
Added patternfly ESLint checking

### PR DESCRIPTION
**Pull request by wojta**
There are ESLint rules for patternfly provided here: 
https://github.com/patternfly/patternfly-react/tree/master/packages/eslint-plugin-patternfly-react

Added checking for them.

### CODE REVIEW

File: ui/.eslintrc.json [hunk 1]
```diff
@@ -9,5 +9,7 @@
     },
     "env": {
         "browser": true
-    }
+    },
+    "plugins": ["plugin:patternfly-react/recommended"]
```
Comment (jhellar) [inline ui/.eslintrc.json]: This should not be in `plugins`, but in `extends` (bug in patternfly's docs). Also the order of `extends` items should be like this:
```
"extends": [
        "airbnb",
        "plugin:patternfly-react/recommended"
    ],
```

**Comment by camilamacedo86** [comment]
Hi Folks, 

I understand that we should use [JavaScript Standard Style](https://standardjs.com/) instead of it. 

c/c @wei-lee @darahayes

File: ui/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (wojta) [reply ui/.eslintrc.js]: ok, possibly, I was following that documentation on the github.

File: ui/.eslintrc.json (hunk 1 above)
Comment (wojta) [reply ui/.eslintrc.json]: +1

**Comment by wojta** [comment]
closing this because of #61

**Pull request closed without merging**